changed twilio for phone, sttThreshold (#167)
bespoken/bespoken-docs: end-to-end/ivr.md

|finishOnPhrase|Phrases that, when detected, will make the test continue to the next utterance.|string, array|Utterance||
|listeningTimeout|The maximum time to listen to before sending the next utterance. Can be used instead of finishOnPhrase.|seconds|Global/Utterance|45|
|recognitionHints|Phrases that improve speech recognition for speech to text detection.|string, array|Utterance||

> |[recordCall](#listening-to-call-recordings)|Whether to record the call. If recorded, the URL for accessing the call will be provided as part of the response in a `callAudioURL` property.|boolean|Global|false|

Reviewer: ```suggestion
|[recordCall](#listening-to-call-recordings)|Whether to record the call. If set to `true`, the URL for accessing the call will be provided as part of the response in the `callAudioURL` property.|boolean|Global|false|
```

|repeatOnPhrase|Repeats the previous utterance when one of these values is found. For cases when the system we are calling does not understand, for whatever reason, our utterance.|string, array|Global/Utterance||
| [runInBand](#test-running-sequence-parallelism) | If set to `true` (default), a test suite will run only when the previous one has finished running. If set to `false` test suites will run in parallel to each other - defaults to `true`|boolean|Global|true|

> | [sttThreshold](#test-running-sequence-parallelism) | A decimal number from 0 to 1 that represents the threshold applied in a fuzzy matching for identifying a `finishOnPhrase` value. Setting this property to 1, means no fuzzy matching is applied. |number|Global|0.8|

Reviewer: ```suggestion
| [sttThreshold](#test-running-sequence-parallelism) | A decimal number from 0 to 1 that represents the threshold applied when using fuzzy matching to identify a `finishOnPhrase` value. Setting this property to 1 means no fuzzy matching is applied. |number|Global|0.8|
```

When this is set, the `recognitionHints` values will be the **only** values sent to Google's speech to text. The more detailed they are, the better the results will be.

### Matching `finishOnPhrase` values

> By default, we apply fuzzy matching when possible on `finishOnPhrase` values to identify the end of an interaction. Fuzzy matching means that we look for a value that is not equal but "similar enough". We do this to bypass ocassional speech to text mismatches that could prevent call interactions from continuing. The confidence level that we use is controlled by the `sttThreshold` property, it allows a numeric value between 0 and 1 . The default value is 0.8, setting it lower will be more forgiving with the transcripts, while setting it to 1 would make the tests look for the exact value that was defined as a `finishOnPhrase`.

Reviewer: ```suggestion
By default, we apply fuzzy matching on `finishOnPhrase` values to identify the end of an interaction. Fuzzy matching means that we look for a value that is not identical but "similar enough" to the supplied value; we do this to bypass ocassional speech-to-text mismatches that could prevent call interactions from continuing. The confidence level that we use is controlled by the `sttThreshold` property, which allows for a numeric value between 0 and 1 . The default value is 0.8; setting it lower is more forgiving with the transcripts, while setting it to 1 makes the tests look for the exact value that was defined as a `finishOnPhrase`.
```
